<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1500</width>
    <height>900</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Maslova Lab09</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="confLayout">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <layout class="QGridLayout" name="colorLayout">
          <item row="0" column="1">
           <widget class="QPushButton" name="colorBtn">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>62</width>
              <height>33</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>62</width>
              <height>33</height>
             </size>
            </property>
            <property name="autoFillBackground">
             <bool>false</bool>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(92, 53, 102);</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="autoExclusive">
             <bool>false</bool>
            </property>
            <property name="autoDefault">
             <bool>false</bool>
            </property>
            <property name="default">
             <bool>true</bool>
            </property>
            <property name="flat">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="colorLbl">
            <property name="minimumSize">
             <size>
              <width>239</width>
              <height>33</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>239</width>
              <height>33</height>
             </size>
            </property>
            <property name="text">
             <string>Цвет:</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="funcLbl">
          <property name="text">
           <string>Функция:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="funcCB">
          <item>
           <property name="text">
            <string>cos(x * sin(z))</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>cos(sqrt(x ^ 2 + z ^ 2))</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>sin(x) ^ 2 + cos(z) ^ 2</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QGridLayout" name="funcLayout">
        <item row="0" column="3">
         <widget class="QLabel" name="stepLbl">
          <property name="text">
           <string>Шаг</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QDoubleSpinBox" name="zToDSB"/>
        </item>
        <item row="1" column="1">
         <widget class="QDoubleSpinBox" name="xFromDSB"/>
        </item>
        <item row="1" column="3">
         <widget class="QDoubleSpinBox" name="xStepDSB"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="xLbl">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>35</width>
            <height>34</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>35</width>
            <height>34</height>
           </size>
          </property>
          <property name="text">
           <string>X:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="toLbl">
          <property name="text">
           <string>До</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QDoubleSpinBox" name="zFromDSB"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="zLbl">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>35</width>
            <height>34</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>35</width>
            <height>34</height>
           </size>
          </property>
          <property name="text">
           <string>Z:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="3">
         <widget class="QDoubleSpinBox" name="zStepDSB"/>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="fromLbl">
          <property name="text">
           <string>От</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QDoubleSpinBox" name="xToDSB"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QGridLayout" name="rotateLayout">
        <item row="1" column="1">
         <widget class="QPushButton" name="yRotateBtn">
          <property name="text">
           <string>по Y</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1" colspan="2">
         <widget class="QDoubleSpinBox" name="rotateDSB"/>
        </item>
        <item row="1" column="2">
         <widget class="QPushButton" name="zRotateBtn">
          <property name="text">
           <string>по Z</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="xRotateBtn">
          <property name="text">
           <string>по X</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="rotateLbl">
          <property name="text">
           <string>Поворот:</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="scaleCoefLbl">
          <property name="text">
           <string>Коэффициент:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QSpinBox" name="scaleCoefSB"/>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QPushButton" name="scaleBtn">
          <property name="text">
           <string>Масштабировать</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="commonLayout">
        <item>
         <widget class="QPushButton" name="drawBtn">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Отобразить</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="clearBtn">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Очистить</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QGraphicsView" name="graphicsView"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
